postprocessing_variant_calls/maf/filter/filter_process.py

```diff
@@ -242,5 +244,63 @@ def cmo_ch(
     return 0
 
 
+@app.command(
+    "access_remove_variants",
+    help="Filter a MAF file based on all the parameters satisfied by the remove variants by annotations CWL script in the ACCESS pipeline",
+)
+def access_remove_variants(
+    maf: Path = typer.Option(
+        ...,
+        "--maf",
+        "-m",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        writable=False,
+        readable=True,
+        resolve_path=True,
+        help="MAF file to subset",
+    ),
+    intervals_file: Path = typer.Option(
+        ...,
+        "--intervals",
+        "-i",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        writable=False,
+        readable=True,
+        resolve_path=True,
+        help="Intervals file containing rows of criterion to tag input MAF by",
+    ),
+    output_maf: Path = typer.Option(
+        "output.maf", "--output", "-o", help="Maf output file name."
+    ),
+    separator: str = typer.Option(
+        "tsv",
+        "--separator",
+        "-sep",
+        help="Specify a seperator for delimited data.",
+        callback=check_separator,
+    ),
+):
+    # prep maf
+    mafa = MAFFile(maf, separator)
+    maf_df = mafa.tag_variant_by_intervals(intervals_file)
```

Comment:
I feel like this should be done in a separate command. Or at least it's not clear from the description that this will also perform tagging. 

I think we need to add a `by_intervals` function as a `tag` sub-command.

Reply:
@buehlere just took a look at this function and I think this was created before we made the decision to just have tag by_variant_annotations and tag access only. since both tag by_variant_annotations and tag access don't rely on this function, I'm not sure if this is needed in the new release.